src/components/ApplicationForm/Skills.js

```diff
@@ -62,22 +62,26 @@ export default ({ formInputs, onChange, role, handleResume }) => {
             <ResumeUploadBtn
               onChange={e => {
                 if (e.target.files[0]) {
-                  onChange({
-                    resume: e.target.value,
-                  })
+                  const size = (e.target.files[0].size / 1024 / 1024).toFixed(2)
```

Comment:
Would it be possible to put this logic under handleResume? I think this would still allow uploads since alert doesn't stop code execution.